pkg/version/version.go

```diff
@@ -31,7 +31,7 @@ var (
 	State       string
 
 	// MaxPluginVersion must be increased to add new plugin and make sure the new plugin version is same to this value
-	MaxPluginVersion = 2
+	MaxPluginVersion = 3
```

Comment:
@vburenin This is requested by @emlin . currently without `rename` getting in, the dataversion of a container is `2`; so any container with dataveresion < 3 should not suppport `rename`.